packages/camera/index.ios.ts

```diff
@@ -143,7 +143,7 @@ export let takePicture = function (options: CameraOptions): Promise<any> {
 		}
 
 		let authStatus = PHPhotoLibrary.authorizationStatus();
-		if (authStatus !== PHAuthorizationStatus.Authorized) {
+		if (authStatus !== PHAuthorizationStatus.Authorized && authStatus !== PHAuthorizationStatus.Limited) {
```

Comment:
It seems that a call to PHPhotoLibrary.authorizationStatus(); will only ever return Authorized, it would need some logic to handle SDK_VERSION >= 14

Reply:
👍